administrator/language/de-DE/com_guidedtours.ini

```diff
@@ -95,7 +95,8 @@ COM_GUIDEDTOURS_TYPE_REDIRECT_URL_DESC="Relative URL der Seite, zu welcher der S
 COM_GUIDEDTOURS_TYPE_REDIRECT_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_DESC="Relative URL der Seite, von der aus die Tour starten soll, z.&#8239;B. administrator/index.php?option=com_guidedtours&view=tours für die Listenansicht der Touren."
-COM_GUIDEDTOURS_USERSTATE_CONNECTEDONLY="Die Aktion „Tour-Benutzerstatus“ ist nur für verbundene Benutzer verfügbar."
-COM_GUIDEDTOURS_USERSTATE_STATENOTSAVED="Tour-Benutzerstatus für Benutzer %1$s Tour %2$s nicht gespeichert."
-COM_GUIDEDTOURS_USERSTATE_STATESAVED="Tour-Benutzerstatus für Benutzer %1$s Tour %2$s gespeichert."
+; 'Tour User states' are conditions in which a guided tour user leaves a tour, either by cancelling, completing or skipping a tour.
+COM_GUIDEDTOURS_USERSTATE_CONNECTEDONLY="Die Aufzeichnung „Tour-Benutzerstatus“ ist nur für verbundene Benutzer verfügbar."
```

Comment:
Hmm Aufzeichnung? Aktion hatte mir besser gefallen oder gab's einen Grund für die Änderung hier den ich nicht sehe?

Reply:
Original: "The recording of User states for a tour is only available to logged-in users."

Da hab ich das Beste draus gemacht. Bin für Vorschläge offen.

Reply:
Reached out to Olivier via Mattermost to confirm the change.